src/audio/eq_iir/eq_iir.c

```diff
@@ -31,30 +31,31 @@
 #include <stdint.h>
 
 /* tracing */
-#define trace_eq(__e, ...) trace_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
-#define trace_eq_with_ids(comp_ptr, __e, ...)			\
+#define trace(__e, ...) trace_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
+#define trace_wi(comp_ptr, __e, ...)			\
 	trace_event_comp(TRACE_CLASS_EQ_IIR, comp_ptr,		\
 			 __e, ##__VA_ARGS__)
 
-#define tracev_eq(__e, ...) tracev_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
-#define tracev_eq_with_ids(comp_ptr, __e, ...)			\
+#define tracev(__e, ...) tracev_event(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
+#define tracev_wi(comp_ptr, __e, ...)			\
 	tracev_event_comp(TRACE_CLASS_EQ_IIR, comp_ptr,		\
 			  __e, ##__VA_ARGS__)
 
-#define trace_eq_error(__e, ...) \
+#define trace_er(__e, ...) \
 	trace_error(TRACE_CLASS_EQ_IIR, __e, ##__VA_ARGS__)
-#define trace_eq_error_with_ids(comp_ptr, __e, ...)		\
+#define trace_ei(comp_ptr, __e, ...)		\
```

Comment:
Same with "ei", I believe you should just leave it as-is.

Reply:
I think "_with_ids" is an ugly bloat. It should be the default trace type and use the original trace macro name. Another macro could be defined for functions without access to "dev" handle.

Reply:
I do agree that it should be the default but i'd take this into a different, more involved discussion rather than a "pass-by refactor". So you should instead make a separate PR where you would do such proposed changes to the traces so we can discuss them rather than blocking this PR with such issues.